contracts/bootstrap/Bootstrap.js

```diff
@@ -81,6 +81,25 @@ class Bootstrap {
 
```

Comment:
everything that's in the bootstrap folder shouldn't change (unless a new sidechain is started) so basically there is no need to add the subscription contract in that folder. (you'll put the sub contract only if you were to start a new sidechain but this contract will be deployed directly on the existing chains)

Reply:
Ok, I'll remove that